examples/freecap/freecap-solver.html

```diff
@@ -0,0 +1,111 @@
+<!doctype html>
+<html>
+<head>
+    <title>CBL-js Example :: freeCap CAPTCHA Solver</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="../../cbl.min.css" />
+    <link rel="stylesheet" type="text/css" href="freecap.css" />
+    <script type="text/javascript" src="../../cbl.min.js"></script>
+</head>
+<body>
+    <div class="main">
+        <div id="captchas">
+          <img id="img-hauaqu" src="../../captchas/freecap/hauaqu.png" />
+          <input type="text" id="hauaqu">
+          <img id="img-luoel" src="../../captchas/freecap/luoel.png" />
+          <input type="text" id="luoel">
+          <img id="img-muqoz" src="../../captchas/freecap/muqoz.png" />
+          <input type="text" id="muqoz">
+          <img id="img-raoon" src="../../captchas/freecap/raoon.png" />
+          <input type="text" id="raoon">
+          <img id="img-xbeea" src="../../captchas/freecap/xbeea.png" />
+          <input type="text" id="xbeea">
+          <img id="img-yoeva" src="../../captchas/freecap/yoeva.png" />
+          <input type="text" id="yoeva">
+          <img id="img-zueme" src="../../captchas/freecap/zueme.png" />
+          <input type="text" id="zueme">
+          <img id="img-hgools" src="../../captchas/freecap/hgools.png" />
+          <input type="text" id="hgools"> 
+          <img id="img-rpoeo" src="../../captchas/freecap/rpoeo.png" />
+          <input type="text" id="rpoeo"> 
+          <img id="img-keoym" src="../../captchas/freecap/keoym.png" />
+          <input type="text" id="keoym">
+          <img id="img-vawmxe" src="../../captchas/freecap/vawmxe.png" />
+          <input type="text" id="vawmxe">
+          <img id="img-qdauu" src="../../captchas/freecap/qdauu.png" />
+          <input type="text" id="qdauu">
+          <img id="img-rbymzo" src="../../captchas/freecap/rbymzo.png" />
+          <input type="text" id="rbymzo">
+          <img id="img-wmecn" src="../../captchas/freecap/wmecn.png" />
+          <input type="text" id="wmecn">
+        </div>
+        <br />
+        <br />
+        <a href="javascript: void(0)" id="solve" onclick="solve()" style="display: none">Solve!</a>
+    </div>
+    <script>
+        var cbl = new CBL({
+            preprocess: function(img) {
+              img.removeGray(80);
+              img.blur(1);
+              img.binarize(255);
+              img.colorRegions(40, true);
+            },
+            /* Load the model we saved during training. */
+            model_file: "freecap-model.txt",
+            character_set: "abcdefghijklmnopqrstuvwxyz",
```

Comment:
I didn't see any example images containing the letters F, I, J, and T. If those are valid for this CAPTCHA, could you provide samples? The model needs to be trained on a sampling of every possible character so it can learn to identify all of them. If they're never used in this system, then they can be removed from the character set.

Reply:
Awesome. Good reference.